content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -249,6 +249,9 @@ Supported options
 {% endnote %}
 
 - `prefix` specifies a prefix for all commit messages.
+   When you specify a prefix for commit messages, {% data variables.product.prodname_dotcom %} will automatically add a colon between the defined prefix and the commit message, unless you end the end the prefix with a space.
```

Comment:
@schra - I have talked with the engineering team and we agreed that we didn't want to document the regexp in the docs. First of all, it doesn't fit in with our content model for the docs, and also it will be a maintenance overhead if/when we may change the implementation. I am therefore suggesting the following:

```suggestion
   When you specify a prefix for commit messages, {% data variables.product.prodname_dotcom %} will automatically add a colon between the defined prefix and the commit message provided the defined prefix ends with a letter, number, closing parenthesis, or closing bracket. This means that, for example, if you end the prefix with a whitespace, there will be no colon added between the prefix and the commit message.
```

Reply:
Sounds good to me!